perception/image_projection_based_fusion/include/image_projection_based_fusion/roi_cluster_fusion/node.hpp

```diff
@@ -44,7 +44,9 @@ class RoiClusterFusionNode
   bool use_cluster_semantic_type_{false};
   float iou_threshold_{0.0f};
   bool remove_unknown_;
+  float thrust_distance_;
```

Comment:
[Q]
What does "thrust" mean? Is it like upper-limit or maximum? Just a typo of "TRUST"?

Reply:
Sorry, @miursh. Fixed it,  it's just a typo.